packages/core/src/components/Range/styles.ts

```diff
@@ -0,0 +1,110 @@
+import { StyleSheet } from '../../hooks/useStyles';
+
+export const HANDLE_SIZE = 18;
+export const HALF_HANDLE_SIZE = HANDLE_SIZE / 2;
+export const TRACK_HEIGHT = 4;
+export const ANNOTATION_SIZE = TRACK_HEIGHT;
+
+export const stylesheetInputRange: StyleSheet = ({ unit, ui, color, font, pattern }) => {
+  const CONTAINER_HEIGHT = 3 * unit;
+
+  return {
+    container: {
+      position: 'relative',
+      height: CONTAINER_HEIGHT,
+    },
+
+    container_disabled: {
+      opacity: 0.4,
+    },
+
+    input: {
+      transform: 'translateY(-2px)', // center-align
+      appearance: 'none',
+      width: '100%',
+      height: TRACK_HEIGHT,
+      margin: 0,
+      background: color.core.neutral[2],
+      outline: 'none',
+      overflow: 'visible',
+      cursor: 'pointer',
+
+      '@selectors': {
+        '::-webkit-slider-thumb': {
+          appearance: 'none',
+          zIndex: 1,
+          transition: `opacity ${ui.transitionTime}, backgroundColor ${ui.transitionTime}`,
+          width: HANDLE_SIZE,
+          height: HANDLE_SIZE,
+          border: `2px solid ${color.accent.borderActive}`,
+          borderRadius: HANDLE_SIZE,
+          backgroundColor: color.accent.bg,
+        },
+
+        ':not([disabled]):active::-webkit-slider-thumb, :not([disabled]):focus::-webkit-slider-thumb': {
+          ...pattern.themedFocus,
+          backgroundColor: color.accent.bgHover,
```

Comment:
sorry, thought we discussed this, but make the border darker too (like inputs and radio button borders get darker on hover/focus). same for hover styles below

Reply:
sorry for the oversight there, done